### DISCUSSION TITLE
How to define aliases in iSH?

### DISCUSSION THREAD

**Discussion by Isaac2k1** [ish-app/ish#1939] [General]
I tried to simply write some aliases in a .profile , but iSH obviously ignores this file. Is there any other means to achieve this functionality?

**Comment by suckerSlayer** [comment]
touch a shell file in the path
"/etc/profile.d/"
for example:
"/etc/profile.d/alpine_shell.sh"

**Reply by Isaac2k1** [reply]
This is not possible: no such file or directory
The Alpine of a-shell runs isolated in a container of iOS and /etc is not accessible. :-(

**Reply by suckerSlayer** [reply]
Emm...btw, it works on mine
I add the following in `/etc/profile`, so it will execute the script in `/etc/profile.d` which contains my aliases.
```sh
for script in /etc/profile.d/*.sh ; do
        if [ -r "$script" ] ; then
                . "$script"
        fi
done
```

**Reply by DeJavLius** [reply]
`/etc/profile` executes all .sh files in `/etc/profile.d/`

so, i added a file named home.sh and edited it with the following code.

```shell
if [ -f ~/.bashrc ]; then
        source ~/.bashrc
fi
``` 

and it works fine for me
